README.md

```diff
@@ -2,6 +2,34 @@
 
 This repository contains GCP connectors for Apache Flink.
 
+## Examples
+
+You can find multiple examples that use GCP connectors on `flink-examples-gcp`.
+
+These are:
+- Google Cloud Storage Load Generator
+- Google Cloud Storage to Google Cloud Storage with Table API
+- Google Cloud Storage to Google Cloud Storage Unbounded WordCount
+- Google Cloud Storage to Google Cloud Storage Bounded WordCount
+- Google Managed Kafka Load Generator
+- Google Managed Kafka to BigQuery
+- Google Managed Kafka to Google Managed Kafka WordCount
+- Google Managed Kafka to Google Managed Kafka with Table API
+- PubSub Load generator
+- PubSub to BigQuery WordCount
+
+## Scripts
+
+Contains scripts to create a Google Manage Flink deployment and start a Flink job.
```

Comment:
the official name if "BigQuery Engine for Apache Flink" instead of Google Managed Flink... i know...